Lua/Comments.tmPreferences

```diff
@@ -15,6 +15,18 @@
 				<key>value</key>
 				<string>-- </string>
 			</dict>
+			<dict>
+				<key>name</key>
+				<string>TM_COMMENT_START_2</string>
+				<key>value</key>
+				<string>--[[</string>
+			</dict>
+			<dict>
+				<key>name</key>
+				<string>TM_COMMENT_END_2</string>
+				<key>value</key>
+				<string>--]]</string>
```

Comment:
It's a somewhat common Lua idiom to use `--]]` to close a block comment like this because it allows you to easily toggle the state of the block comment by adding an extra `-` to the `--[[` token. It's mentioned in PiL for example: https://www.lua.org/pil/1.3.html